test/Website/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM microsoft/dotnet:1.0-runtime
+
```

Comment:
I think so, for maximum immutability. If dotnet released a new patch version which introduced a bug, having dynamic base image may make the issue hard to trace.